velox/dwio/parquet/reader/PageReader.cpp

```diff
@@ -700,6 +700,16 @@ void PageReader::makeDecoder() {
         break;
       }
       FMT_FALLTHROUGH;
```

Comment:
This is needed for `case Encoding::DELTA_BYTE_ARRAY:` to go to `default:`.
This change breaks that.
Move `case Encoding::RLE:` before `case Encoding::DELTA_BYTE_ARRAY:`